feconf.py

```diff
@@ -452,7 +457,7 @@ def get_empty_ratings():
 DASHBOARD_STATS_DATETIME_STRING_FORMAT = '%Y-%m-%d'
 
 # The maximum size of an uploaded file, in bytes.
-MAX_FILE_SIZE_BYTES = 1048576
+MAX_FILE_SIZE_BYTES = 4048576
```

Comment:
Why?

Reply:
This was by mistake! (I was testing some issue while writing the commit message.) Removed!